src/content/learn/writing-markup-with-jsx.md

```diff
@@ -166,7 +166,7 @@ img { height: 90px }
 </>
 ```
 
-इस खाली टैग को *[React fragment](TODO)* कहा जाता है। React fragments आपको ब्राउज़र HTML ट्री में कोई निशान छोड़े बिना चीजों को ग्रुप करने देते हैं।
+इस खाली टैग को *[React fragment](https://react.dev/reference/react/Fragment)* कहा जाता है। React fragments आपको ब्राउज़र HTML ट्री में कोई निशान छोड़े बिना चीजों को ग्रुप करने देते हैं।
```

Comment:
can you update the link similar to we have in english version of the doc? https://github.com/reactjs/react.dev/blob/main/src/content/learn/writing-markup-with-jsx.md?plain=1#L172